tests/e2e/java/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's not clear why we need a new test module. Why cant the tests be integration tests without the existing SDK module?

Reply:
Reverted.